README.md

```diff
@@ -121,8 +125,8 @@ export let authenticator = new Authenticator(sessionStorage, { sessionKey: '_ses
 
 authenticator.use(new GoogleStrategy(
   {
-    clientID: "YOUR_CLIENT_ID",
-    clientSecret: "YOUR_CLIENT_SECRET",
+    clientID: "107822922492186567430",
+    clientSecret: "3594d0aae2394dbd501258ca5585437ef27e0d5c",

```

Comment:
Any reason you've set these?

Reply:
:face_palm:
I was testing dummy credentials. I have removed it.